hera.rb

```diff
@@ -0,0 +1,69 @@
+class Hera < Formula
+  desc "Hera: Ewasm virtual machine conforming to the EVMC API"
+  homepage "https://github.com/ewasm/hera"
+
+  stable do
+    url "https://github.com/ewasm/hera/archive/v0.2.4.tar.gz"
+    sha256 "6d1bb30764c892f1aaada26c08add021a981d1d27638d2c9e198ca831544f9d1"
+
+    resource "cmake/cable" do
+      url "https://github.com/ethereum/cable.git", tag: "v0.2.17"
+    end
+
+    resource "evmc" do
+      url "https://github.com/ethereum/evmc.git", tag: "v6.2.2"
+    end
+  end
+
+  head "https://github.com/ewasm/hera.git"
+
+  option "with-debugging", "Build with debugging features and messages"
+  option "without-shared-libraries", "Build libraries as static"
+  option "with-binaryen", "Build with binaryen"
+  option "without-wabt", "Build without wabt"
+  option "with-wavm", "Build with WAVM"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@6"
+
+  def install
```

Comment:
Wondering if `install` should check whether at least one of the engines is present. Currently that check is made on the cmake stage, but we could potentially show a better error.